### ISSUE TITLE
Cannot drag on Android Chrome

### ISSUE THREAD

**Issue by azmartone** [asgordon/rotodrag-js#1]
Device: Samsung Galaxy s10 Android Chrome
![image](https://user-images.githubusercontent.com/262556/107829787-5288db00-6d3f-11eb-8e35-f2fe82a49f99.png)

Seems to be broken on Android Chrome. 

Current Behavior:
Dragging just causes the page to scroll up and down

Expected Behavior: Able to drag the SVG children without page moving up and down

**Comment by asgordon** [comment]
I was able to reproduce this problem on a recent ChromeBook, although I noticed that double-tapping the shape allowed me to drag it as expected.
I was able to get it to work correctly on the ChromeBook by adding "touch-action:none;" to the css of the svg, e.g. changing docs.css to style svg objects as follows:

svg {
    width: 100%;
    touch-action:none;
}

I'll investigate this further to see if there might be a better solution. Thanks-
